man/doc2tex.pl

```diff
@@ -327,6 +341,7 @@
 	prolog_id_cont(CL),
 	{ atom_codes(Name, [C0|CL]) }.
 
+prolog_id_cont(_) --> "(", !, { fail }.
```

Comment:
Why is this?   `(` is not a Prolog id continue character, no?

Reply:
This is a left-over from when I was using your original ordering of `tr//2` rules.
I ended up re-ordering them, so that the _identifier(...)_ stuff is done before detecting an atom -- otherwise, things such as `PlTerm::unify_blob()` weren't handled correctly.
It appears to be unnecessary (and a cut-fail isn't good style either).